build-tools/scripts/y2autoconf

```diff
@@ -288,7 +288,7 @@ PKG_CHECK_EXISTS([yast2-core],[
 ])
 
 CFLAGS="${CFLAGS} -Wall -Wformat=2 -Wmissing-prototypes"
-CXXFLAGS="${CXXFLAGS} ${Y2CORE_CFLAGS} -Wall -Wformat=2"
+CXXFLAGS="${CXXFLAGS} ${Y2CORE_CFLAGS} -Wall -Wformat=2 -Wno-format-nonliteral"
```

Comment:
Is it is safe to enable it globally? Do the other packages (the ruby-bindings, pkg-bindings,...) still build with this option?

Reply:
They will build, because the added option *disables* one warning included in Wformat=2.

But this gives me an idea to use `osc` to ask for all the build logs and get all the currently produced warnings.

Reply:
Ah, I see...